prow_config.yaml

```diff
@@ -52,3 +52,45 @@ workflows:
       - kubeflow/tf-serving/*
       - components/k8s-model-server/*
       - testing/*
+
+  # Image Auto Release workflows.
+  # The workflows below are related to auto building our Docker images.
```

Comment:
Should we delete the current release cron job if prow will take over?

Reply:
We should but lets do that in a follow on PR.